src/shared/database/amendments.ts

```diff
@@ -1,5 +1,9 @@
 import axios from 'axios'
-import createHash from 'create-hash'
+import { Client, ErrorResponse } from 'xrpl'
+import {
+  FeatureAllResponse,
+  FeatureOneResponse,
+} from 'xrpl/dist/npm/models/methods/feature'
```

Comment:
Are these not exported by `xrpl` normally?

Reply:
No it is not being exported in index file. We'll need to fix that in xrpl.js. I'll add a TODO

Reply:
I have made changes to resolve this